examples/change_attribute/change_attitude.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+© Copyright 2017, Team OpenMakerDrone at Yonsei Univ.
+
+change_attitude.py: (Copter Only)
+
+This example shows how to move/direct Copter and send commands in GUIDED_NOGPS mode using DroneKit Python.
+
+"""
+
+from dronekit import connect, VehicleMode, LocationGlobal, LocationGlobalRelative
+from pymavlink import mavutil # Needed for command message definitions
+import time
+import math
+
+#Set up option parsing to get connection string
+import argparse  
+parser = argparse.ArgumentParser(description='Control Copter and send commands in GUIDED mode ')
+parser.add_argument('--connect', 
+                   help="Vehicle connection target string. If not specified, SITL automatically started and used.")
+args = parser.parse_args()
+
+connection_string = args.connect
+sitl = None
+
+#Start SITL if no connection string specified
+if not connection_string:
+    import dronekit_sitl
+    sitl = dronekit_sitl.start_default()
+    connection_string = sitl.connection_string()
+
+
+# Connect to the Vehicle
+print 'Connecting to vehicle on: %s' % connection_string
+vehicle = connect(connection_string, wait_ready=True)
+
+def arm_and_takeoff_nogps(aTargetAltitude):
+    """
+    Arms vehicle and fly to aTargetAltitude without GPS data.
+    """
+    
+    ##### CONSTANTS #####
+    DEFAULT_TAKEOFF_THRUST = 0.7
+    SMOOTH_TAKEOFF_THRUST = 0.6
+    
+    print "Basic pre-arm checks"
+    # Don't let the user try to arm until autopilot is ready
+    # If you need to disable the arming check, just comment it with your own responsibility.
+    while not vehicle.is_armable:
+        print " Waiting for vehicle to initialise..."
+        time.sleep(1)
+
+
+    print "Arming motors"
+    # Copter should arm in GUIDED_NOGPS mode
+    vehicle.mode = VehicleMode("GUIDED_NOGPS")
+    vehicle.armed = True
+
+    while not vehicle.armed:
+        print " Waiting for arming..."
+        time.sleep(1)
+
+    print "Taking off!"
+
+    thrust = DEFAULT_TAKEOFF_THRUST
+    while True:
+        current_altitude = vehicle.location.global_relative_frame.alt
+        print " Altitude: ", current_altitude
+        if current_altitude >= aTargetAltitude*0.95: # Trigger just below target alt.
+            print "Reached target altitude"
+            break
+        elif current_altitude >= aTargetAltitude*0.6:
+            thrust = SMOOTH_TAKEOFF_THRUST
+        set_attitude(thrust = thrust)
+        time.sleep(0.2)
+
+
+def set_attitude(roll_angle = 0.0, pitch_angle = 0.0, yaw_rate = 0.0, thrust = 0.5, duration = 0):
```

Comment:
If you don't add a `yaw_angle` argument here, the drone will always abruptly flip to that yaw angle as soon as you call this function.

In my instance of using this code I initialize the `yaw_angle` that I'm passing to this function to the drone's current yaw heading for it to work smoothly.